src/containers/MoodleIntegrationConfig/index.jsx

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
```

Comment:
I think I removed this on my branch, but if not I should.

This redux container only gets triggered in the enterpriseApp page and since we are only referencing LmsConfigurations, Moodle/CanvasIntegrationConfig containers are useless. So for those they should just involve standard prop passing from LmsConfigurations (which I know I recently did for Moodle)

Reply:
Point here was I don't think we need the Canvas container either.